### PR TITLE
fix:select multiple rows

### PR DESCRIPTION
**Pull request by shaonianche**
image

### CODE REVIEW

**Comment by yihong0618** [comment]
could you explain how to reproduce this bug?

**Comment by shaonianche** [comment]
> could you explain how to reproduce this bug?

https://user-images.githubusercontent.com/16186646/189293790-08dac8c4-0897-4f59-a5d5-b709b7b93031.mov

Click on the list on the right, then click on the left. Then repeat ...

File: src/components/RunTable/RunRow.jsx [hunk 1]
```diff
@@ -15,14 +15,13 @@ const RunRow = ({ runs, run, locateActivity, runIndex, setRunIndex }) => {
...
-    if (runIndex !== -1 && elementIndex !== runIndex) {
-      elements[runIndex].style.color = MAIN_COLOR;
+    for (let i = 0; i < elements.length; i += 1) {
+      elements[i].style.color = MAIN_COLOR
     }
-    setRunIndex(elementIndex);
+    const elementIndex = runs.indexOf(run)
+    e.target.parentElement.style.color = 'red'
+    setRunIndex(elementIndex)
   };
```
Comment (yihong0618) [inline src/components/RunTable/RunRow.jsx:25]: do we have some better solution
this need to  loop the elements every time?

Comment (shaonianche) [reply src/components/RunTable/RunRow.jsx:25]: My girlfriend said yes, but she couldn't tell me until the weekend ~

Comment (yihong0618) [reply src/components/RunTable/RunRow.jsx:25]: I figure out some better solution, will push use your `PR`

**Comment by yihong0618** [comment]
@shaonianche fixed please review

**Review by yihong0618 — APPROVED** [review]

**Pull request merged**